src/helpers/formulaHelper.js

```diff
@@ -0,0 +1,187 @@
+const formulas = require('./formulas.json')
+
+const TOKEN_TYPE = {
+    Number: Symbol("number"),
+    Variable: Symbol("variable"),
+    Function: Symbol("function"),
+    Operator: Symbol("operator")
+};
+
+const FUNCTION_TOKENS = [
+    "MIN",
+    "MAX",
+    "ROUND"
+];
+
+function getFormulaByKey(key) {
+    return formulas[key];
+}
+
+function parse(formulaObj) {
+    const constants = formulaObj.constants;
+    let tokens = formulaObj.formula.split('');
+
+    let resultStack = [];
+    let parseStack = [];
+
+    for (let i = 0; i < tokens.length; i++) {
+        if (tokens[i] == ' ')
+            continue;
+        
+        if (tokens[i] >= '0' && tokens[i] <= '9') {
```

Comment:
Let's extract these into constants too

Reply:
Wait, do you mean extracting `0` and `1` into constants?

Reply:
Yes, or rather how do you make it clear what you mean. What is `0` and `9` exactly? Maybe instead of constants, extract it into a function?

Reply:
Yeah, I'd rather change it into a function than making into a constant, because this expression literally means check if the character is a numeric character. It checks whether the ascii value of the character is between 48 (character `0`) or 57 (character `9`).

Reply:
Sure, change it into a function like that, e.g. `isNumeric`